src/test/java/seedu/address/storage/JsonAdaptedPersonTest.java

```diff
@@ -134,11 +133,18 @@ public void toModelType_invalidTags_throwsIllegalValueException() {
 
     @Test
     public void toModelType_invalidBirthday_throwsIllegalException() {
```

Comment:
Might want to split this into 2 separate test cases since you are testing 2 different errors